ext/tags.go

```diff
@@ -88,6 +88,18 @@ var (
 
 	// Error indicates that operation represented by the span resulted in an error.
 	Error = boolTagName("error")
+
+	//////////////////////////////////////////////////////////////////////
+	// PerfEvent Tag
+	//////////////////////////////////////////////////////////////////////
+
+	// PerfEvent records the list of events given in a comma separated string (e.g.
+	// "cpu-cycles,cache-misses,branch-misses") for a span.
+	// The tracer is expected to start the events in PerfEvent using the "perfevents"
+	// package (https://github.com/opentracing-contrib/perfevents/go) . On span
+	// Finish(), the events must be closed.
+
+	PerfEvent = stringTagName("perfevents")
```

Comment:
@yurishkuro I went through the PRs (uber/jaeger-client-go#94 and https://github.com/uber/jaeger-client-go/pull/103 ) regarding an "observer" interface. I did a small prototype implementation for the perfevents package as well, defining the Observer and SpanObserver API in opentracing-go, an observer implementation in the perfevents package and used zipkin to register the oberservers. While this model worked great for starting the perfevents (onSetTag), reading the values and closing the event descriptors (onFinish) and maintaining the count (in the SpanObserver struct), we got issues while logging these event values for a span . The reason being, we couldn't access the values in the (client) tracer or in a sample app which creates the perfevents observer. The perf event values are per-span. Could you please point out if we missed anything obvious?